cryptoki/src/types/object.rs

```diff
@@ -27,6 +28,8 @@ pub enum AttributeType {
     Decrypt,
     /// Determines if it is possible to derive other keys from the key
     Derive,
+    /// Parameters defining an elliptic curve.
```

Comment:
This is extremely nitpicky but other doc comments did not end with the dot (`. `). Maybe it'd be worth to keep it consistent one way or another?

Reply:
Sure, will do, I prefer following these conventions - in this case I just wasn't paying attention 😅